custom-commands/scripts/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "resourceName": {
+      "value": "test7resource"
+    },
+    "luisName": {
+      "value": "test7resource-8070"
+    },
+    "functionName": {
+      "value": "test7resource-8070"
+    },
+    "storageName": {
+      "value": "test7resource8070"
+    }
+  }
+}

```

Comment:
Please see one of my recent changes. I updated test7resource to generic name.

Reply:
See https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/blob/master/custom-commands/hospitality/deployment/azuredeploy.parameters.json

Reply:
OK. I will update azuredeploy.parameters.json accordingly.